RELEASES.md

```diff
@@ -1,3 +1,152 @@
+Version 1.45.0 (2020-07-16)
+==========================
+
+Language
+--------
+- [Float to int conversion using `as` has been defined as a saturating conversion.][71269]
+  This was previously undefined behaviour.
+- [`mem::Discriminant<T>` now uses `T`'s discriminant type instead of always
+  using `u64`.][70705]
+- [Procedural macros can now be used in expression, pattern, and  statement
+  positions.][68717] This now means you can now use a procedural macro anywhere
+  you can use a declarative (`macro_rules!`) macros.
+
+Compiler
+--------
+- [You can now override individual target features through the `target-feature`
+  flag.][72094] E.g. `-C target-feature=+avx2 -C target-feature=+fma` is now
+  equivalent to `-C target-feature=+avx2,+fma`.
+- [Added the `force-unwind-tables` flag.][69984] This option allows
+  rustc to always generate unwind tables regardless of panic strategy.
+- [Added the `embed-bitcode` flag.][71716] This codegen flag allows rustc
+  to include LLVM bitcode into generated `rlib`s (this is on by default).
+- [Added the `tiny` value to the `code-model` codegen flag.][72397]
+- [Added tier 3 support\* for the `mipsel-sony-psp` target.][72062]
+- [Added tier 3 support for the `thumbv7a-uwp-windows-msvc` target.][72133]
+
+\* Refer to Rust's [platform support page][forge-platform-support] for more
+information on Rust's tiered platform support.
+
+
+Libraries
+---------
+- [`convert::Infallible` now implements `Error`.][71633]
```

Comment:
I think this is a bit misleading.
#71633 changed Error impl of `Infallible` to implement Error directly (no new implementation has been added, previously `Infallible` was implementing `Error` via type alias). It actually only affects the document.